lib/cfa/augeas_parser.rb

```diff
@@ -2,6 +2,9 @@
 require "forwardable"
 require "cfa/placer"
 
+require "cfa/augeas_parser/keys_cache"
+require "cfa/augeas_parser/writer"
```

Comment:
I know and I consider it, but problem is that AugeasParser is already class, so I cannot have it as module. but on other hand I want to point that it is internal classes of AugeasParser, so that is reason for that name.